chapters/theoretical_foundations/sections/non-eudlidean-spaces/subsections/analytic_description.tex

```diff
@@ -0,0 +1,34 @@
+\subsection{Analytic description}
+To describe the non-Euclidean geometries analytically, we will follow the approach given by \cite{Szirmay-Kalos2022}.
```

Comment:
I think we should add the name of the work. Right now on PDF it just shows "given by [3]"

Reply:
I think that's ok, I've seen it written like that in other papers